conbench/api/_google.py

```diff
@@ -1,44 +1,72 @@
 import json
+import logging
 import os
+import time
 
 import flask as f
 import requests
 
+from ..config import Config
 
-def get_google_config():
+log = logging.getLogger(__name__)
+
+# Rely on INTENDED_BASE_URL to have a trailing slash.
+OIDC_CALLBACK_URL = Config.INTENDED_BASE_URL + "api/google/callback"
+
+
+def get_oidc_config():
     client_id = os.environ.get("GOOGLE_CLIENT_ID", None)
     client_secret = os.environ.get("GOOGLE_CLIENT_SECRET", None)
-    discovery_url = "https://accounts.google.com/.well-known/openid-configuration"
+    discovery_url = Config.OIDC_ISSUER_URL + "/.well-known/openid-configuration"
     return discovery_url, client_id, client_secret
 
 
-def get_google_client():
+def get_oidc_client():
     from oauthlib.oauth2 import WebApplicationClient
 
-    discovery_url, client_id, _ = get_google_config()
-    google = requests.get(discovery_url).json()
+    discovery_url, client_id, _ = get_oidc_config()
+
+    # Pragmatic healing for transient errors. Better: cache OP config across
+    # login requests. Also, if we want to retry here in the future: consider
+    # using tenacity.
```

Comment:
This is super minor, and I don't really mind either way since this doesn't sound quite like a thing we will definitely do in the future, but I like calling TODOs out with `# TODO:` since many IDEs flag those | makes them easier to search.  Up to you if you think using tenacity is really a to-do or if it's more a "if we have issues with this code, we might consider"

Reply:
>  I like calling TODOs out with # TODO

Good to know. Me, too! :)  In this case indeed I just wanted to quickly dump my thoughts, it's basically a "think about this again" todo item and because it's so unspecific and not so important I found a TODO label a little too fat.